Dockerfile

```diff
@@ -14,7 +8,7 @@ LABEL org.kennethreitz.vendor="Kenneth Reitz"
 ENV LC_ALL=C.UTF-8
```

Comment:
Since this is basically a copy and paste from the original repo, I didn't see problems in letting the original author here, but eventually, we can set Kong as the author.